### PR TITLE
Fix ng build visibility issues

### PR DESCRIPTION
**Pull request by calbach**

### CODE REVIEW

File: ui/src/app/job-list/job-list.component.html [hunk 1]
```diff
@@ -4,5 +4,5 @@
   [pageSize]="pageSize"></jm-header>
 <jm-job-list-table
   [dataSource]="dataSource"
-  (onJobsChanged)="handleJobsChanged(jobs)">
+  (onJobsChanged)="handleJobsChanged($event)">
```
Comment (alahwa) [inline ui/src/app/job-list/job-list.component.html]: Doesn't handleJobsChanged still accept a QueryJobsResult[]? This change seems unnecessary given the one below

Comment (calbach) [reply ui/src/app/job-list/job-list.component.html]: Correct. The emitter just sends the subset of jobs that were modified (not all jobs). I gather that this parameter needs to be specially called `$event`.

That said, the job list component doesn't actually even look at this information currently. Just seemed like a reasonable thing to emit.

Comment (alahwa) [reply ui/src/app/job-list/job-list.component.html]: In that case, would it be better to just remove that parameter entirely?

Comment (calbach) [reply ui/src/app/job-list/job-list.component.html]: Yeah, good point. I've left the emitter value, but dropped this callback configuration.

**Pull request merged**